src/com/owncloud/android/ui/errorhandling/ErrorMessageAdapter.java

```diff
@@ -55,267 +56,121 @@
 
 public class ErrorMessageAdapter {
 
-    public ErrorMessageAdapter() {
-    }
+    private static class Formater {
+        final Resources r;
+        Formater(Resources r) {
+            this.r = r;
+        }
 
+        public String f(final int m) {
```

Comment:
👍 what about using **format**?

Reply:
Hm still not a fan of it.

Reply:
Would you keep `f.f` or what's your more readable alternative? I suggested you that way because is self-explanatory and is being used in other Java 7 formatters: https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html

Reply:
I would have kept `f.f` yes, but I understand the reason why we want to go self-explanatory.